README.md

```diff
@@ -144,6 +144,12 @@ With Chocolatey installed, execute the following command to install posh-git:
 choco install poshgit
 ```
 
+### Installing post-git Manually
+
+If you need to test/debug changes prior to contrubiting here, or would otherwise prefer to install post-git without the aid of a package manager you can run `Import-Module POST-GET-PATH` where `POST-GET-PATH` is the path to the `src\posh-git.psd1` file.
+
+Say you have posh-git located at `$home\documents\github\posh-git`, you can install posh-git by running `Import-Module $home\documents\github\posh-git\src\posh-git.psd1`.
```

Comment:
I wouldn't say "install posh-git by running".  Better to say "import posh-git by executing".